### ISSUE TITLE
No Hash entry in Release file 

### ISSUE THREAD

**Issue by rs-mrichmond** [sensu/sensu#1210]
The repo is currently broken in Ubuntu 16.04.

Apt gives the following error after installing to repo and trying to update
`W: http://repositories.sensuapp.org/apt/dists/sensu/Release.gpg: 
Signature by key EE15CFF6AB6E4E290FDAB681A20F259AEB9C94BB uses weak digest algorithm (SHA1)
E: Failed to fetch http://repositories.sensuapp.org/apt/dists/sensu/Release  
No Hash entry in Release file /var/lib/apt/lists/partial/repositories.sensuapp.org_apt_dists_sensu_Release 
which is considered strong enough for security purposes
 `

This isn't currently a big issue but at the end of the month when people start migrating from 14.04 to 16.04 it will likely cause headaches if it isn't fixed. :(

**Comment by rs-mrichmond** [comment]
Note: This might look like a duplicate of issue #1135 because I believe they are both frieght related

**Comment by calebhailey** [comment]
@rs-mrichmond I think you're right that this is a duplicate. We finally triaged that issue yesterday and tagged it for resolution in the 0.24 packages (though it's possible that we'll fix this before we release 0.24). I'll go ahead and close this in favor of #1135, but feel free to comment here again if you disagree.

**Comment by peterlavelle** [comment]
Hi,

Having an issue installing the sensu repo. I changed the repository source from  repos.sensuapp.org to repositories.sensuapp.org but the sensu package is flagged as un-authenticated when I attempt to install it. 

Output from apt-key list is below

```
pub   2048R/7580C77F 2012-05-02
uid                  Sonian Inc (Sensu signing key) <chefs@sonian.net>

pub   2048R/EB9C94BB 2015-08-31
uid                  Freight <support@hw-ops.com>
```

I'm using http://repositories.sensuapp.org/apt/keyring.gpg  as the source URL for the key.
